includes/REST/AdminNoticeController.php

```diff
@@ -36,6 +37,14 @@ public function register_routes() {
                     'methods'             => WP_REST_Server::READABLE,
                     'callback'            => [ $this, 'dokan_get_admin_notices' ],
                     'permission_callback' => [ $this, 'check_permission' ],
+                    'args'                => [
+                        'scope' => [
+                            'description' => __( 'Notice context', 'dokan-lite' ),
+                            'type'        => 'string',
+                            'required'    => false,
+                            'default'     => 'all',
+                        ],
```

Comment:
Change the default value.

Reply:
vai, default value changed with your feeback